hotels.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>Отели в Седоне</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=PT+Sans:wght@400;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css" integrity="sha512-NhSC1YmyruXifcj/KFRWoC561YpHpc5Jtzgvbuzx5VozKpWvQ+4nXhPdFgmx8xqexRcpAglTj9sIBWINXa8x5w==" crossorigin="anonymous" />
+    <link href="css/style.css" rel="stylesheet">
+  </head>
+  <body class="page-body">
+    <header class="main-header">
+      <nav class="main-navigation">
+        <a class="main-header-logo">
+          <img src="img/logo.svg" width="138" height="70" alt="Логотип Седона">
+        </a>
+        <ul class="site-navigation">
+          <li><a href="">Информация</a></li>
+          <li><a href="">Фото и видео</a></li>
+          <li><a href="">Карта штата</a></li>
+          <li><a href="hotels.html">Гостиницы</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class=" container">
+
+      <section class="filter-search-hotels">
+
+        <form class="filter-form" method="post" action="https://echo.htmlacademy.ru/" oninput="priceoutput.value=pricecount.value">
+          <fieldset>
+            <legend>Инфраструктура:</legend>
+              <ul>
+                <li class="filter-checkbox filter-range">
+                  <input type="checkbox" name="swimingpool" id="filter-swimingpool" checked>
+                  <label for="filter-swimingpool">Бассейн</label>
+                </li>
+                <li class="filter-checkbox filter-range">
+                  <input type="checkbox" name="parking" id="filter-parking">
+                  <label for="filter-parking">Парковка</label>
+                </li>
+                <li class="filter-checkbox filter-range">
+                  <input type="checkbox" name="wi-fi" id="filter-wi-fi">
+                  <label for="filter-wi-fi">Wi-Fi</label>
+                </li>
+              </ul>
+          </fieldset>
+          <fieldset>
+            <legend>Тип жилья:</legend>
+              <ul>
+                <li class="filter-checkbox filter-range">
+                  <input type="checkbox" name="hotel" id="filter-hotel" checked>
+                  <label for="filter-hotel">Гостиница</label>
+                </li>
+                <li class="filter-checkbox filter-range">
+                  <input type="checkbox" name="motel" id="filter-motel" checked>
+                  <label for="filter-motel">Мотель</label>
+                </li>
+                <li class="filter-checkbox filter-range">
+                  <input type="checkbox" name="appartament" id="filter-appartament" checked>
+                  <label for="filter-appartament">Аппартаменты</label>
```

Comment:
```suggestion
                  <label for="filter-appartament">Апартаменты</label>
```

Пример того, как делать кастомные чекбоксы https://www.w3schools.com/howto/tryit.asp?filename=tryhow_css_custom_checkbox